### PR TITLE
Add plugin display name

### PR DESCRIPTION
**Pull request by mzazrivec**
See ManageIQ/manageiq-ui-classic#4496 for primary motivation.

### CODE REVIEW

File: lib/manageiq/api/engine.rb [hunk 1]
```diff
@@ -34,6 +34,10 @@ def self.log_kv(key, val)
       def vmdb_plugin?
         true
       end
+
+      def self.plugin_name
+        _('API')
```
Comment (bdunne) [inline lib/manageiq/api/engine.rb]: Should this be more specific like `REST API`?

Comment (mzazrivec) [reply lib/manageiq/api/engine.rb]: Sure, why not. Fixed.

**Comment by bdunne** [comment]
@abellotti or @gtanzillo Can you merge?

**Comment by abellotti** [comment]
Thanks @mzazrivec for enhancing this.  👍

**Pull request merged**